pkg/webhook/preflight/nutanix/storagecontainer.go

```diff
@@ -20,7 +20,7 @@ const (
 )
 
 type storageContainerCheck struct {
-	nodeSpec *carenv1.NutanixNodeSpec
+	nodeSpec *carenv1.NutanixMachineDetails
```

Comment:
Lets also rename `nodeSpec` var?

Reply:
Done, renamed the field to "machineSpec".